test-clients/src/main/java/com/hedera/services/bdd/suites/ethereum/EthereumSuite.java

```diff
@@ -0,0 +1,91 @@
+package com.hedera.services.bdd.suites.ethereum;
+
+import com.hedera.services.bdd.spec.HapiApiSpec;
+import com.hedera.services.bdd.suites.HapiApiSuite;
+import com.hedera.services.ethereum.EthTxData;
+import com.hederahashgraph.api.proto.java.ResponseCodeEnum;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.List;
+
+import static com.hedera.services.bdd.spec.HapiApiSpec.defaultHapiSpec;
+import static com.hedera.services.bdd.spec.assertions.AssertUtils.inOrder;
+import static com.hedera.services.bdd.spec.assertions.ContractFnResultAsserts.resultWith;
+import static com.hedera.services.bdd.spec.assertions.TransactionRecordAsserts.recordWith;
+import static com.hedera.services.bdd.spec.keys.KeyFactory.KeyType.THRESHOLD;
+import static com.hedera.services.bdd.spec.queries.QueryVerbs.getAccountBalance;
+import static com.hedera.services.bdd.spec.queries.QueryVerbs.getTxnRecord;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.contractCreate;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.cryptoCreate;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.cryptoTransfer;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.ethereumCall;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.uploadInitCode;
+import static com.hedera.services.bdd.spec.transactions.crypto.HapiCryptoTransfer.tinyBarsFromAccountToAlias;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.newKeyNamed;
+
+public class EthereumSuite extends HapiApiSuite {
+
+	private static final Logger log = LogManager.getLogger(EthereumSuite.class);
+	private static final long depositAmount = 20_000L;
+
+	private static final String PAY_RECEIVABLE_CONTRACT = "PayReceivable";
+
+	public static void main(String... args) {
+		new EthereumSuite().runSuiteSync();
+	}
+
+	@Override
+	public List<HapiApiSpec> getSpecsInSuite() {
+		return List.of(
+				invalidNonceEthereumTxFails()
+		);
+	}
+
+	HapiApiSpec invalidNonceEthereumTxFails() {
+		return defaultHapiSpec("InvalidNonceEthereumTxFails")
+				.given(
+						newKeyNamed(SECP_256K1_SOURCE_KEY).shape(SECP_256K1_SHAPE),
+						cryptoCreate(RELAYER).balance(6 * ONE_MILLION_HBARS),
+						cryptoTransfer(tinyBarsFromAccountToAlias(GENESIS, SECP_256K1_SOURCE_KEY, ONE_HUNDRED_HBARS)),
+
+						uploadInitCode(PAY_RECEIVABLE_CONTRACT),
+						contractCreate(PAY_RECEIVABLE_CONTRACT).adminKey(THRESHOLD)
+				).when(
+						ethereumCall(PAY_RECEIVABLE_CONTRACT, "deposit", depositAmount)
+								.type(EthTxData.EthTransactionType.EIP1559)
+								.signingWith(SECP_256K1_SOURCE_KEY)
+								.payingWith(RELAYER)
+								.via("payTxn")
+								.nonce(1l)
```

Comment:
I think the first nonce used for a new account should be 0.

Reply:
Yes, it should. That's why I assert that the status is WRONG_NONCE. This is failing test case.